examples/aei_from_events/aei_from_events.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+all_cat = ['festivals', 'school-holidays', 'airport-delays', 'sports',
```

Comment:
If all categories is basically the sum of the other three categories, you can get it by summing them.

```
NON_ATTENDED_CATEGORIES = [...]
ATTENDED_CATEGORIES = [...]
UNSCHEDULED_CATEGORIES = [...]

ALL_CATEGORIES = NON_ATTENDED_CATEGORIES + ATTENDED_CATEGORIES + UNSCHEDULED_CATEGORIES
```

Reply:
This is exactly as I have done! Perfect!